packages/gatsby-theme-aio/src/components/Layout/index.js

```diff
@@ -108,9 +108,15 @@ export default ({ children, pageContext, location }) => {
       (async () => {
         try {
           await addScript(`${IMS_SRC}`);
-          window.adobeImsFactory.createIMSLib(JSON.parse(IMS_CONFIG));
+          let IMS_CONFIG_JSON = JSON.parse(IMS_CONFIG);
+
+          IMS_CONFIG_JSON.onReady = function() {
```

Comment:
This would override any given `onReady` function provided in the config. Ideally, we would check if there's already a provided `onReady` function and monkey patch it.

Reply:
Ah ok I shall make the edit!